test/src/test/resources/fixtures/namespaced/typename-is-resolved-even-when-namespaced-type-is-extended-in-declaring-service.yml

```diff
@@ -0,0 +1,122 @@
+name: "typename is resolved even when namespaced type is extended in declaring service"
```

Comment:
These tests are duplicates of existing tests with the execution hint turned on.

The only difference is that the `__typename` has been removed from the expected service query and result.